### PR TITLE
update sidecars to latest version

### PR DESCRIPTION
**Pull request by humblec**
> /kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: deploy/kubernetes/storage-capacity.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccount: csi-provisioner
       containers:
         - name: csi-provisioner
-          image: registry.k8s.io/sig-storage/csi-provisioner:v3.1.0
+          image: registry.k8s.io/sig-storage/csi-provisioner:v3.3.0
```
Comment (msau42) [inline deploy/kubernetes/storage-capacity.yaml:25]: Can we make these templates so we don't need to remember to update the images every release?

Comment (humblec) [reply deploy/kubernetes/storage-capacity.yaml:25]: will open an issue for the improvement/automation @msau42 .

Comment (humblec) [reply deploy/kubernetes/storage-capacity.yaml:25]: @msau42 can we get this in regardless?

**Pull request merged**